ci-operator/config/openshift/installer/openshift-installer-master__altinfra-periodics.yaml

```diff
@@ -141,6 +141,215 @@ tests:
       TEST_TYPE: conformance-parallel
     workflow: openshift-e2e-powervs-ipi
   timeout: 6h0m0s
+- as: e2e-openstack-fips
+  minimum_interval: 56h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+      FIPS_ENABLED: "true"
+      TEST_SUITE: experimental/reliability/minimal
+    pre:
+    - chain: ipi-openstack-pre
+    - ref: fips-check
+    workflow: openshift-e2e-openstack-ipi
```

Comment:
This altinfra image is not fips compliant so you may need to set `OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION="true"` to skip the installer host validation. We could do that in a followup pr once we observe failures.